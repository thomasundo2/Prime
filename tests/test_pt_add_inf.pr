int main()
{
    /* creation and assignment */
    pt p1;
    pt p2;
    pt p3;
    lint m1;
    curve crv;

    m1 = 1l - 2l;

    crv = [(5l, 12l) : 13l];
    p1 = [m1, m1] & crv;
    p2 = [7l, 0l] & crv;
    printpt(p2 + p1);
    p3 = p2 + p1;
    printpt(p3);
    printpt([m1, m1] & [(5l, 12l) : 13l] + [7l, 0l] & [(5l, 12l) : 13l]);
    printpt([7l, 0l] & [(5l, 12l) : 13l] + [m1, m1] & [(5l, 12l) : 13l]);

    return 0;
}

