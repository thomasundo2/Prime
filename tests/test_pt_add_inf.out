[7,0] & [(5,12) : 13]
[7,0] & [(5,12) : 13]
[7,0] & [(5,12) : 13]
[7,0] & [(5,12) : 13]
